app/models/assignment.rb

```diff
@@ -9,7 +9,7 @@ class Assignment < ActiveRecord::Base
   include AssignmentAnalytic
   include ReviewAssignment
   include QuizAssignment
-  include OnTheFlyCalc
+  include AssignmentScoreHelper
```

Comment:
put these methods into assignment_helper.rb

Reply:
New commit should do it